### PR TITLE
add swin_unetr pretrain repository

### PR DESCRIPTION
**Pull request by ahatamiz**
Signed-off-by: ahatamizadeh <ahatamizadeh@nvidia.com>

This PR adds the repository for pre-training of Swin UNETR model as described in the accepted CVPR paper [1]. 

[1]: Tang, Yucheng, Dong Yang, Wenqi Li, Holger Roth, Bennett Landman, Daguang Xu, Vishwesh Nath, and Ali Hatamizadeh. "Self-supervised pre-training of swin transformers for 3d medical image analysis." arXiv preprint arXiv:2111.14791 (2021).

### CODE REVIEW

File: SwinUNETR/Pretrain/README.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+The following datasets were used for pre-training (~5050 3D CT images):
+
+- Head & Neck Squamous Cell Carcinoma (HNSCC) ([Link](https://wiki.cancerimagingarchive.net/display/Public/HNSCC))
+- Lung Nodule Analysis 2016 (LUNA 16) ([Link](https://luna16.grand-challenge.org/Data/))
+- TCIA CT Colonography Trial ([Link](https://wiki.cancerimagingarchive.net/display/Public/CT+COLONOGRAPHY/))
+- TCIA Covid 19 ([Link](https://wiki.cancerimagingarchive.net/display/Public/CT+Images+in+COVID-19/))
+- TCIA LIDC ([Link](https://wiki.cancerimagingarchive.net/display/Public/LIDC-IDRI/))
+
+Please see the corresponding the json files of each dataset in ```jsons``` folder.
```
Comment (wyli) [inline SwinUNETR/Pretrain/README.md:28]: I uploaded the jsons to https://github.com/Project-MONAI/MONAI-extra-test-data/releases/tag/0.8.1,

please use links such as https://github.com/Project-MONAI/MONAI-extra-test-data/releases/download/0.8.1/dataset_HNSCC_0.json  and remove the json files from this PR.

Comment (ahatamiz) [reply SwinUNETR/Pretrain/README.md]: @wyli Thanks. Addressed.

Comment (wyli) [reply SwinUNETR/Pretrain/README.md]: could you please also remove the content in the `json` folder? in general the data are not archived in this repo.

Comment (ahatamiz) [reply SwinUNETR/Pretrain/README.md]: sounds good

**Review by wyli — APPROVED** [review]

**Pull request merged**